Schedule issue for monthly web application vulnerability scan (#7100) (#7121)
DataBiosphere/azul: .github/ISSUE_TEMPLATE/web_app_vulnerability_scan.md

---
name: Run the web app vulnerability scans

> about: Issue template for the monthly web app vulnerability scans and triaging of the findings

Reviewer: I'd shorten a bit as …
```suggestion
about: Issue template for the monthly scanning and triaging of web app vulnerabilities
```

title: Monthly web app vulnerability scans
labels: +,compliance,infra,no demo,orange

> assignees: dsotirho-ucsc

Reviewer: I know you've been the point person for the task, but consider confirming this. It may be easier to detect an unassigned issue than an assigned one.

_repository: DataBiosphere/azul-private
_start: 2025-06-01T09:00
_period: 1 month
---

### Author

- [ ] Completed an authenticated ZAP vulnerability scan of …
  - [ ] … the HCA Data Portal https://data.humancellatlas.org/
  - [ ] … the HCA Data Browser https://explore.data.humancellatlas.org/
  - [ ] … the HCA Azul Indexer https://indexer.azul.data.humancellatlas.org/
  - [ ] … the HCA Azul Service https://service.azul.data.humancellatlas.org/
  - [ ] … the AnVIL Data Portal https://anvilproject.org/
  - [ ] … the AnVIL Data Browser https://explore.anvilproject.org/
  - [ ] … the AnVIL Azul Indexer https://indexer.explore.anvilproject.org/
  - [ ] … the AnVIL Azul Service https://service.explore.anvilproject.org/
- [ ] Uploaded scan report PDFs to the [Google Drive](https://drive.google.com/drive/folders/1LYg9qV6usZT20vxddWjSkONmrw9-PjHy)

> - [ ] Added findings to the [Google Sheet](https://docs.google.com/spreadsheets/d/1uUaiOGr8t2oKSDal_FazIUDBhNb1yq3RwbSy-xMMBO0/edit?gid=709622447#gid=709622447) and created issues for new `High`, `Medium`, and `Low` severity findings

Reviewer: Is a bit more duplicative but more adequate for the CL.
```suggestion
- [ ] Added findings to the [Google Sheet](https://docs.google.com/spreadsheets/d/1uUaiOGr8t2oKSDal_FazIUDBhNb1yq3RwbSy-xMMBO0/edit?gid=709622447#gid=709622447) and created …
  - [ ] … issues for new `High` severity findings
  - [ ] … issues for new `Medium` severity findings
  - [ ] … issues for new `Low` severity findings
```

- [ ] Added a comment to issues previously created for newly fixed findings (e.g. "No longer a finding as of the YYYY-MM-DD ZAP scan")
- [ ] Added a comment to this issue listing the newly created issues, a count of fixed findings, and a screenshot of the Google Sheet

> - [ ] Assigned issue to only the system administrator

Reviewer: Like our other CLs.
```suggestion
- [ ] Issue is assigned to only the system administrator
```

  - [ ] … the HCA Azul Indexer https://indexer.azul.data.humancellatlas.org/
  - [ ] … the HCA Azul Service https://service.azul.data.humancellatlas.org/
  - [ ] … the AnVIL Data Portal https://anvilproject.org/
  - [ ] … the AnVIL Data Browser https://explore.anvilproject.org/
  - [ ] … the AnVIL Azul Indexer https://indexer.explore.anvilproject.org/
  - [ ] … the AnVIL Azul Service https://service.explore.anvilproject.org/
- [ ] Uploaded scan report PDFs to the [Google Drive](https://drive.google.com/drive/folders/1LYg9qV6usZT20vxddWjSkONmrw9-PjHy)
- [ ] Added the findings to the [Google Sheet](https://docs.google.com/spreadsheets/d/1uUaiOGr8t2oKSDal_FazIUDBhNb1yq3RwbSy-xMMBO0/edit?gid=709622447#gid=709622447), and …
  - [ ] … created issues for new `High` severity findings
  - [ ] … created issues for new `Medium` severity findings
  - [ ] … created issues for new `Low` severity findings
- [ ] Added a comment to issues previously created for newly fixed findings (e.g. "No longer a finding as of the YYYY-MM-DD ZAP scan")
- [ ] Added a comment to this issue listing the newly created issues, a count of fixed findings, and a screenshot of the Google Sheet

> - [ ] Issue is assigned to only the system administrator
> - [ ] Moved issue to `Review Requested`

Reviewer: Apologies, I originally missed this. For consistency with our other CLs.
```suggestion
- [ ] Moved issue to `Review Requested`
- [ ] Issue is assigned to only the system administrator
```
